djangocms_versioning/monkeypatch/extensions.py

```diff
@@ -0,0 +1,26 @@
+from cms.extensions.extension_pool import ExtensionPool
+from cms.models.titlemodels import PageContent
+
+
+def _copy_title_extensions(self, source_page, target_page, language, clone=False):
+    # the line below has been modified to accomodate versioning.
+    # import pdb; pdb.set_trace()
+    source_title = PageContent._original_manager.filter(
```

Comment:
Should we not be using _base_manager vs _original_manager?

Reply:
Does not work for _base_manager. We need the _original_manager.